MAINTAINERS.md

```diff
@@ -1,16 +1,24 @@
+## Maintainers
+
+### Active Maintainers
 | Name | GitHub | RocketChat |
 | --- | --- | --- |
 | Adam Ludvik | aludvik | adamludvik |
```

Comment:
Adam appears in both active and Retired maintainers?

Reply:
whoops, he should not still be in active, updated